ds-card-reader/device/physical.go

```diff
@@ -90,12 +91,18 @@ func (reader *CardReaderPhysical) Status() error {
 func (reader *CardReaderPhysical) Write(commandName string, cardNumber string) {
 	// assemble the values that will be propagated throughout the
 	// device service
+	commandvalue, err := dsModels.NewCommandValueWithOrigin(
+		commandName,
+		edgexcommon.ValueTypeString,
+		cardNumber,
```

Comment:
Maybe resource name `card-data` in the device profile should be `card-number`. Would be a better descriptive name.